CoreFoundation/URL.subproj/CFURLSessionInterface.h

```diff
@@ -48,6 +48,10 @@ typedef struct CFURLSessionEasyCode {
     int value;
 } CFURLSessionEasyCode;
 
+CF_EXPORT CFStringRef _Nonnull CFURLSessionErrorDescription(int value);
+
+CF_EXPORT int const CFURLSessionEasyErrorSize;
```

Comment:
This feels like an abstraction leak, but I'm not sure what to do about it exactly.

Reply:
yeah, the buffer thing is a bit untidy.